packages/typespec-client-generator-core/test/decorators.test.ts

```diff
@@ -21,7 +21,13 @@ import {
   shouldGenerateConvenient,
   shouldGenerateProtocol,
 } from "../src/decorators.js";
-import { SdkMethodResponse, SdkOperationGroup, UsageFlags } from "../src/interfaces.js";
+import {
+  SdkClientType,
+  SdkHttpOperation,
+  SdkMethodResponse,
+  SdkOperationGroup,
```

Comment:
seems we will not use `SdkOperationGroup` in `sdkPackage.clients`, do we still need this import?

Reply:
this import is for other tests of `@operationGroup` decorator.

Reply:
ok, for `listClients()`?

Reply:
we still have the `@operationGroup` decorator so we still need to support it with tests